builders/debian11.Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM debian:bullseye
+
+RUN apt update && apt install -y \
+    wget build-essential git tree software-properties-common dirmngr apt-transport-https ufw libssl-dev libpcre2-dev zlib1g-dev
```

Comment:
Previously some builds used wget to manually get zlib and pcre libraries used by nginx and this is simpler

Reply:
This is a great update!